lib/plugin.js

```diff
@@ -4,33 +4,52 @@ const _ = require('lodash')
 const routes = require('./routes')()
 const adapterUtils = require('./utils/adapter')()
 const routeUtils = require('./utils/route')()
+const includes = require('./includes');
+
+// constants
+const MIME = {
+    standard: ['application/vnd.api+json', 'application/json'],
```

Comment:
Right, initially I've copied that together with `sendResponse` method. But then I've removed it.
I think that sooner or later we need to add error handling to send them in proper format.